src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/ThrowHelper.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+#nullable enable
+
+namespace Microsoft.Data.SqlClientX
+{
+
+    static class ThrowHelper
```

Comment:
I don't really understand the why behind this class. To me it seems equally unverbose to use a `throw new` versus call a method like this. 
Some of the exception classes also add `ThrowIf` static methods to them, too

Reply:
Forgot to clean this one up. I don't think we have a need for it.